coordinator/testing/pom.xml

```diff
@@ -407,5 +409,75 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>it-dse-next</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <artifactId>maven-failsafe-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>dse-next</id>
+                <goals>
+                  <goal>integration-test</goal>
+                  <goal>verify</goal>
+                </goals>
+                <configuration>
+                  <systemPropertyVariables>
+                    <!-- The ccm.version is just for display purposes,
+                     the ccm.directory determines the actual distribution used -->
+                    <ccm.version>4.0.7</ccm.version>
```

Comment:
minor (optional): If this is just for the display purpose, can we add something to say that this is `dse-next`?

Reply:
not to the property itself. this value is passed to the CcmBridge class from the Java Driver, which checks that it is a valid version number.